js/apps/system/_admin/aardvark/APP/frontend/js/arango/arangoValidationHelper.js

```diff
@@ -0,0 +1,143 @@
+/* jshint unused: false */
+(function () {
+  "use strict";
+
+  window.arangoValidationHelper = {
+    getControlCharactersRegex: () => {
+      return /[^\u0000-\u001F]/;
```

Comment:
Is this regex actually correct?
IMO it matches already if a _single_ character is outside the control characters class, but it doesn't check the entire string.
Example:
```
> "foobar\tab".match(/[^\u0000-\u001F]/)
[ 'f', index: 0, input: 'foobar\tab', groups: undefined ]

> "foobar\tab".match(/^[^\u0000-\u001F]+$/)
null
```